pandas/core/generic.py

```diff
@@ -3607,6 +3607,10 @@ def head(self, n=5):
         """
         Return the first n rows.
 
+        This function returns rows for the object based on the underlying
```

Comment:
Might be good to use 'returns the first n rows' instead of just 'returns rows'.

Reply:
Yup will update.